app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for active_relationships(current_user).build, remote: true do |f| %>
```

Comment:
It seems to me that, you haven't resolved this issue with the way I referred. 
1. In the users_helper.rb file, create a method like as follows
```ruby
def build_active_relationships
  current_user.active_relationships.build 
end 
```
2. Then call it from the view
```ruby
<%= form_for build_active_relationships, remote: true do |f| %>
```